Changes dribbler rule text (#3)
robocup-ssl-br/rules-ssl-el: chapters/robots.adoc

A robot must fit inside a 0.18 meters wide and 0.15 meters high cylinder at any point in time. Additionally, the top of the robot must adhere to the standard pattern size and surface constraints.

==== Dribbling Device

> The dribbling device, present in SSL division B and A, will not be allowed in SSL-EL. Robots can have the device, but it will not be allowed activated during the game.

Reviewer: ```suggestion
The dribbling device present in the Regular League will not be allowed in the SSL-EL. Robots can have the device, but it will not be allowed to be activated during the game.
```
